docs_nnx/why.rst

```diff
@@ -64,15 +66,19 @@ Here's an example of a simple Flax NNX program that illustrates many of the poin
 
     return loss
 
-Improvements
-------------
-Through the rest of this document, we'll key examples of how Flax NNX improves on Flax Linen.
+Flax NNX's improvements on Linen
+--------------------------------
+
+The rest of this document uses various examples that demonstrate how Flax NNX improves on Flax Linen.
 
 Inspection
 ^^^^^^^^^^
-The first improvement is that Flax NNX modules are regular Python objects, so you can easily
-construct and inspect them. Because Flax Linen Modules are lazy, some attributes are not available
-upon construction and are only accesible at runtime. This makes it hard to inspect and debug.
+
+The first improvement is that Flax NNX Modules are regular Python objects. This means that you can easily
+construct and inspect ``Module`` objects.
+
+Compare this to Flax Linen, where Modules are lazy, which means some attributes are not available upon construction
+and are only accessible at runtime, making Linen not easy to inspect and debug.
```

Comment:
:+1:  Done.

```
The first improvement is that Flax NNX Modules are regular Python objects. This means that you can easily
construct and inspect ``Module`` objects.

On the other hand, Flax Linen Modules are not easy to inspect and debug because they are lazy, which means some attributes are not available upon construction and are only accessible at runtime.
```